frontend/jupyter/src/app/pages/index/index-default/config.ts

```diff
@@ -5,87 +5,78 @@ import {
   ActionIconValue,
   ActionButtonValue,
   MenuValue,
-  DialogConfig,
   ComponentValue,
   TableConfig,
-  TABLE_THEME,
   DateTimeValue,
+  LinkValue,
+  LinkType,
 } from 'kubeflow';
 import { ServerTypeComponent } from './server-type/server-type.component';
+import { quantityToScalar } from '@kubernetes/client-node/dist/util';
 import { ProtBComponent } from './protb-icon/protb-icon.component';
 
-// --- Configs for the Confirm Dialogs ---
-export function getDeleteDialogConfig(name: string): DialogConfig {
-  return {
-    title: $localize`Are you sure you want to delete this notebook server? ${name}`,
-    message: $localize`Warning: Your data might be lost if the notebook server
-                       is not backed by persistent storage`,
-    accept: $localize`DELETE`,
-    confirmColor: 'warn',
-    cancel: $localize`CANCEL`,
-    error: '',
-    applying: $localize`DELETING`,
-    width: '600px',
-  };
-}
-
-export function getStopDialogConfig(name: string): DialogConfig {
-  return {
-    title: $localize`Are you sure you want to stop this notebook server? ${name}`,
-    message: $localize`Warning: Your data might be lost if the notebook server
-                       is not backed by persistent storage.`,
-    accept: $localize`STOP`,
-    confirmColor: 'primary',
-    cancel: $localize`CANCEL`,
-    error: '',
-    applying: $localize`STOPPING`,
-    width: '600px',
-  };
-}
-
 // --- Config for the Resource Table ---
 export const defaultConfig: TableConfig = {
+  dynamicNamespaceColumn: true,
   columns: [
     {
       matHeaderCellDef: $localize`Status`,
       matColumnDef: 'status',
       value: new StatusValue(),
+      sort: true,
     },
     {
       matHeaderCellDef: $localize`Name`,
       matColumnDef: 'name',
       style: { width: '25%' },
-      value: new PropertyValue({
-        field: 'name',
-        tooltipField: 'name',
+      value: new LinkValue({
+        field: 'link',
+        popoverField: 'name',
         truncate: true,
+        linkType: LinkType.Internal,
       }),
+      sort: true,
     },
     {
       matHeaderCellDef: '',
       matColumnDef: 'prot-b',
       value: new ComponentValue({
         component: ProtBComponent,
       }),
+      sort: true,
     },
     {
       matHeaderCellDef: $localize`Type`,
       matColumnDef: 'type',
       value: new ComponentValue({
         component: ServerTypeComponent,
       }),
-    },
-    {
-      matHeaderCellDef: $localize`Age`,
+      sort: true,
+      sortingPreprocessorFn: element => element.serverType,
+      filteringPreprocessorFn: element => {
+        if (element.serverType === 'group-one') {
+          return 'vscode Visual Studio Code';
+        } else if (element.serverType === 'group-two') {
+          return 'rstudio';
+        } else {
+          return 'jupyterlab';
```

Comment:
If they mentioned the two groups  + jupyterlab, do we want to add our third group to this option or wait...

Reply:
oh you are correct! Ill go update that now